shuttle/rpc/rpc.go

```diff
@@ -239,7 +238,8 @@ func (m *manager) SendRPCMessage(ctx context.Context, handle string, cmd *rpceve
 		m.log.Debugf("sending rpc message: %s, to shuttle: %s using websocket engine", cmd.Op, handle)
 		return d.SendMessage(ctx, cmd)
 	}
-	return websocketeng.ErrNoShuttleConnection
+	m.log.Warnf("failed to sending rpc message: %s, to shuttle: %s - shuttle is offline", cmd.Op, handle)
```

Comment:
no need to fail here

Reply:
is this assuming that any RPC message we send has a retry mechanism elsewhere, and none of the callers would want to error handle?

i'm thinking of the consolidation commands where we would want to know that the message failed to send so we can mark the zone as not-consolidating anymore. otherwise it wouldn't retry because it's perpetually "in consolidation" already

Reply:
@neelvirdy it is retried. I will revert this part, we can tackle it later.